### PR TITLE
[WIP] Gsgmf 680 output message on upgrade

### PR DESCRIPTION
**Pull request by julsbreakdown**
https://jira.camptocamp.com/browse/GSGMF-680

I have also added the demo options, when used this will print out the specific commands used for the demo build => https://github.com/camptocamp/c2cgeoportal/wiki/New-release#in-demo

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 1]
```diff
@@ -673,7 +676,7 @@ def step11(self, step):
             os.unlink("create.diff")
 
         check_call(["make", "--makefile=" + self.options.new_makefile, "build"])
-
+        self.options.use_makefile = True
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: This is wrong

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 2]
```diff
@@ -692,7 +695,12 @@ def step11(self, step):
                 "docker-compose down; docker-compose up -d`.",
                 "- Test your application on 'http://localhost:8480/desktop'."
             ]
-
+        if self.options.demo:
+           message.append(
+               "Please run the following:"
+               "./docker-run rm /build/c2ctemplate-cache.json"
+               "./docker-run --home make --makefile=<user>.mk docker-compose.yaml"
+           )
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: What's this hack?

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py [hunk 3]
```diff
@@ -743,6 +751,8 @@ def step13(self, _):
         print("git push {0!s} {1!s}.".format(
             self.options.git_remote, branch
         ))
+        if self.options.demo:
+            print('Now execute ./scripts/publish <instance>')
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: This shouldn't pollute the standard upgrade

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py (hunk 2 above)
Comment (julsbreakdown) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: https://github.com/camptocamp/c2cgeoportal/wiki/New-release#in-demo

File: geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py (hunk 1 above)
Comment (julsbreakdown) [reply geoportal/c2cgeoportal_geoportal/scripts/c2cupgrade.py]: Yes (should have put WIP).

**Pull request closed without merging**